src/_Problems_/find-2nd-max/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * You may find it easy but it's tricky for few
+ * Input - [9, 2, 3, 6]
+ * Output - 6
+ */
+
+function findSecondMax(arr) {
+  let max = arr[0];
+  let max2 = 0;
```

Comment:
Initialize max2 = -1. If there are no Second Max then function should return -1.
Input : [1,1,1]
Output : -1

Reply:
I think initializing and returning ```null``` makes more sense in case of **JavaScript**

Reply:
Oh yes, I find something better

```javascript 
let max2 = Number.MIN_SAFE_INTEGER;
```